scst-store/cluser-specific-scanner-configurations.hbs.md

```diff
@@ -0,0 +1,42 @@
+# Connecting Vulnerability scanning to SCST - Store 
+This page summarizes how to configure and connect your scanner with Supply Chain Security Tools - Store and provides links to more detailed explanations of individual topics.
+
+## Metadata Store
+ - Single cluster - Scanners will use `app-tls-cert` to communicate with SCST - Store
+ - Multicluster … Sacnners will use `ingress-cert` of SCST - Store in view cluster. Please refer [MultiCluster Setup](multicluster-setup.hbs.md)
+
+
+## Grype and other supported scanner configuration
+The configuration to connect the Grype scanner or another supported scanner to SCST - Store lies in the scanner configuration. 
+
+### Option 1: Single cluster configuration
+In a single cluster configuration, the connection between the scanning pod and the metadata store exists inside the cluster and does not pass through ingress. An ingress connection to the store is not needed.
+
+The default values automatically configure the connection between the Grype scanner or another supported scanner to SCST - Store. You do not need to make any changes to the `grype` section of the `tap-values.yaml` provided in the [full profile installation](../install.hbs.md#install-profile).   
+
+```console
```

Comment:
do we really want to include this here since it's duplicative with the content in the link above?